wagtail_localize_rws_languagecloud/test/settings.py

```diff
@@ -183,3 +183,11 @@
 # we will set up the settings we need for each test case
 # using the @override_settings decorator
 SILENCED_SYSTEM_CHECKS = ["wagtail_localize_rws_languagecloud.E002"]
+
+# Import settings from local_settings.py file if it exists. Please use it to
+# keep settings that are not meant to be checked into Git and never check it
+# in.
+try:
+    from .local_settings import *  # noqa
+except ImportError:
+    pass
```

Comment:
Sneaking this in here if I could. I use `local_settings` test explicitly set `DATABASES['default']['TEST']['name']` to something, because otherwise tests are run on an in memory database and `--keepdb` doesn't work.

Reply:
Having used this pattern elsewhere, I'm OK with it

Reply:
Been thinking of this. Do we need to document it in the readme at all?

Reply:
Good call. I've added a note in the README.